presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java

```diff
@@ -110,6 +107,7 @@
     private static final String LT_OP = "$lt";
     private static final String LTE_OP = "$lte";
     private static final String IN_OP = "$in";
+    @SuppressWarnings("unused")
     private static final String NOTIN_OP = "$nin";
```

Comment:
You can just remove NOTIN_OP instead of `@SuppressWarnings` annotation.

Reply:
Sorry, my mistake.